### ISSUE TITLE
[Integration] Language around deployments changed in OCP4.5 and breaks tests

### ISSUE THREAD

**Issue by jsliacan** [code-ready/crc#1255]
### General information

  * OS: Linux + macOS + Windows
  * Hypervisor: KVM + Hyper-V + hyperkit

## CRC version
```bash
crc version: 1.10.0+b4745a8
OpenShift version: 4.5.0-0.nightly-2020-05-26-011640 (not embedded in binary)
```
  
## CRC status
```bash
CRC VM:          Running
OpenShift:       Running (v4.5.0-0.nightly-2020-05-26-011640)
Disk Usage:      12.38GB of 32.72GB (Inside the CRC VM)
Cache Usage:     9.272GB
Cache Directory: /home/jsliacan/.crc/cache
```

### Steps to reproduce

  1. Start CRC and do `oc login`.
  2. Create project (or not)
  3. Create app: `oc new-app centos/httpd-24-centos7~https://github.com/sclorg/httpd-ex`
  4. Check the status with `oc rollout status dc/httpd-ex`

### Expected
```
deployment "httpd-ex" successfully rolled out
```

### Actual
```
Error from server (NotFound): deploymentconfigs.apps.openshift.io "httpd-ex" not found
```

**Comment by jsliacan** [comment]
Suggestion would be to probe the rollout status with `oc rollout status deployment.apps/httpd-ex` instead. Can't find any documentation saying that this is equivalent though.

**Comment by jsliacan** [comment]
Solved by #1325
